docs/concepts/07_dip/03_consumer.md

```diff
@@ -0,0 +1,88 @@
+---
+id: consumer-pallet
+title: Consumer pallet
+---
+
+This pallet is a component of the DIP protocol.
+It enables entities with an identity on a connected Substrate-based chain (provider) to use those identities on the chain this pallet is deployed (consumers) without requiring those entities to set up a new identity locally.
+
+A consumer chain is _connected_ to a provider if there is a way for the consumer chain to verify state proofs about parts of the state of the provider chain.
+
+A cross-chain transaction with DIP assumes the entity submitting the transaction has already generated a cross-chain identity commitment on the provider chain, by interacting with the DIP provider pallet on the provider chain.
+With a generated identity commitment, a cross-chain transaction flow for a generic entity `A` works as follows:
+
+1. `A` generates a state proof proving the state of the identity commitment on the provider chain.
+2. `A` generates any additional information required for the consumer runtime to successfully verify an identity proof.
+3. `A`, using their account `AccC` on the consumer chain, calls the `dispatch_as` extrinsic by providing its identifier on the provider chain, the generated proof, and the `Call` to be dispatched on the consumer chain.
+
+    1. This pallet verifies if the proof is correct, if not it returns an error.
+    2. This pallet dispatches the provided `Call` with a new origin created by this pallet, returning any errors the dispatch action returns. The origin contains the information revealed in the proof, the identifier of the acting subject and the account `AccC` dispatching the transaction.
+
+    <!-- TODO: Mermaid diagram? -->
+
+The pallet is agnostic over the chain-specific definition of _identity proof verifier_ and _identifier_, although, when deployed, you must configure them to respect the definition of identity and identity commitment established by the provider linked to this pallet.
+
+For instance, if the provider establishes that an identity commitment is a Merkle root of a set of public keys, an identity proof for the consumer will most likely be a Merkle proof revealing a subset of those keys.
+Similarly, if the provider defines an identity commitment as some ZK commitment, the respective identity proof on the consumer chain will be a ZK proof verifying the validity of the commitment and the revealed information.
+
+For identifiers, if the provider establishes that an identifier is a public key, the consumer pallet must use the same definition.
+You must configure other definitions for an identifier, such as a simple integer or a [Decentralized Identifier (DID)](https://www.w3.org/TR/did-core/) in the same way.
+
+The pallet allows the consumer runtime to define some `LocalIdentityInfo` associated with each identifier, which the pallet's proof verifier can access and optionally modify upon proof verification.
```

Comment:
Would it be good to have a navigation back the definition of the `LocalIdentityInfo` or a info box here?